sdk/core/Azure.Core/tests/AzureEventSourceTests.cs

```diff
@@ -39,7 +39,7 @@ void LogEvent(EventSource azureCoreEventSource)
             try
             {
                 List<EventWrittenEventArgs> events = new();
-                using var listener = new AzureEventSourceListener((args, s) => events.Add(args), EventLevel.Verbose);
```

Comment:
can we keep a test verifying that we do the formatting when the other constructor overload is used?

Reply:
There are several in the existing file "AzureEventSourceListenerTests.cs" (not touched in this PR)